lib/ui/views/home/home_view.dart

```diff
@@ -76,32 +75,14 @@ class _HomeViewState extends State<HomeView> {
                 backgroundColor: Colors.white,
                 title: Row(
                   children: <Widget>[
-                    Chip(
-                      label: Padding(
-                        padding: EdgeInsets.symmetric(horizontal: 10),
-                        child: Text(
-                          'Restaurants',
-                          style: TextStyle(
-                              fontFamily: 'Lato',
-                              color: Colors.white,
-                              fontSize: 14,
-                              fontWeight: FontWeight.w600),
+                    Wrap(
+                      children: [
+                        SelectorChip('Restaurants', Mode.Restaurants, model),
```

Comment:
any reason why viewmodel is passed in as argumetn? 

if selector chip is used only in homeview, then it can just extends ViewModelWidget<HomeModel>

Reply:
@alvindrakes that is another way sure, both are references not instantiations right